### ISSUE TITLE
Updating 3D Tiles style doesn't always update existing tiles

### ISSUE THREAD

**Issue by mramato** [CesiumGS/cesium#8618]
The above image has a single 3D Tiles tileset being styled with a single green/peach style.

![image](https://user-images.githubusercontent.com/1495196/74750146-4441d500-5239-11ea-84fa-628ad37d9ddf.png)

The differently styled data blue/white/beige data is the previous style I had applied which Cesium failed to update.

I don't have a reproducible test case yet, but it looks like it has to do with cancelling a camera flight mid-style change.  Trying to write up a Sandcastle now.

**Comment by mramato** [comment]
Okay, here you go: [Sandcastle](https://sandcastle.cesium.com/index.html#c=xVhtb9s2EP4rhFHAMuDo3ZGcptnSuF0D9A2JsaGo94GWaJkoRbokZccL8t93lGRVduw2TTNMHxKJupfn7p6jjnYcdI5SkgskZohyTSRONF0SFIzQmDKikNJrRnk24Y6Drqt7xDDPCpwRNBJJkROusaaCn6C51gt14jgZ1fNiaicid845ZmtNE8z+kHgxp4m65IkTpEfaWHe0JMTJsQLHjlqQhM5A1Bhzrlt+XxaUpcZxijVGiSikJmptIL//dIE+LAgfmRcLITVmFQxAsVqtPJuvEzsTS0dRTZxUUK0dyqmm2ESpDJBpbdxeQEQTvsQSLSlZEYleIE5W6IIoWuT2n+Wa1U3KxwsBQVNOZLePbiccwQUhSFj6KMWSpkSebBQTSbAmfwnJ0nElYvX6lYokXwui9BXhpYKWBQC46z2fcMBR+rNVQjixMyamxE7JQs/HoHCegRWla3OA02iWWqZIRCM9J6gKk6EE51DVMiakBWJCfEFYo3eYz7HWmFch19IfBSQK0g82a/gX2CSbYh7YMynyEcmgZMo6ikLb9eLYC1w/dl0/jPsodO3joeeFQRBEXhj7Xh9Fg8CE0/LwQdINYbYT/IZgU4aPVCfzK8HYljvfs/0ojoZxFPvDIA4GfXQEa0EYDN3BwI0jPzwGd67tRt7xcOANAdowDr1B6b2dyiodtiLaVNSqi5dCVimvabyTi7pY4hvwkz3B1FJQyrHEXM2EzBsKvMNa0pvQvhy9ej++HH9qigzleitwWtbLUHnDRYXK9iDarnJXP20nrPoXjMbVS+sWFZI1Pi8FvyIKWiUhZSbPFchcptYgCr0euruXl4WkOS2bwsZpatUeG5gXggnZwjfFiqQIagjQqUTwJ5sbtLOCJ2VtE6Pwcv2mfGH1mi6p4zLbCjkcj+l+simOuUpzJ6i1Uq3ytKyROkGft1+Z63P32W2F7A6dvUCB60K7dmU2xVYIDHINPw1pBr3u3/0fqvuNuuW5PugC4byB9yBdr6UbgV/vGAz4bvgg5UGj6/shqPnH8CeIHwZ60OgOjMvANX7dB4JudE17exH4jaOHQW6iHRpNFzS9g5kyu1cj70d9NBiCI5DeFm493lW3JYvvDjF0XXOTQX/qIiXAFiFT0+VkD1Hf1lK/TtWUzODLoO6RdYPjCvY5bBjbldWd9ex28+6u121F2X8C/u94Lavj2pFXEmPSKe1a3UUhF4x0e5POoeIetNM2I0n6GBveFhYB+2f2KCxe28yaMCZWjzHjbsFhNH8UGLdtJFljfthI3QCN9JQVlctfbYCUKvjCJwSZLwDCZnBLcyy/7OH/qBZ9Mv6je2xt0AD1N/fWkiS+1WoEwbO6E/poX3/AcqlyBBPB0IuiaOCXH35Ttsg9PvbD3nd7CH2nidDBLtrAheqaIcM1tG/KlUm8/h5DdpXduCx2Tm+sHa72UbuR+shqKx+VynEPOZWV8GdchrsuzVzFWx6/QdjjFHz9nNfTStjd6sjVHIbwB3RBC2XVCQ3IBnTbVxvYLzXMCo4t0CTdoAvzXf3x4DAu7umW97B8RTJy80STDTBv0rEkyV7dLCwA0LNh6NbWNcyNPIOCGBjA/R76DbV3FEP8YQ+Ut7dxs+xBOvbEfD0XKxjb2LoVd1aeUGDKnGNuhgOUE3hUJgn357o5HG6ebqxTBg5sB63hoZyzdpFfw8aVwDGRETOdjoVgUyxfFloLbnVfQzRjAVFvQFq92sW9b/vzav2/P/HsePkfTj0lgj0nnxlbj8X2fv3Ds8/Dzz8/cQb6VuEN3EPXD+p/YfYBhl4zw6B9NNiXhqRUqnSsBsWE75xbnnf6ndOS1WeVrd9pbn5nMGcty7YdTfIFfJyIcqZF8gU6IFHKGDp1NkqnKV0imr6AbXD7p4NJByUMKwVvZgVj1/QfMumcnTogv6XGREmRD0siGV4bkbl39rZatG371IHH+1q6ylHL4r8)

1. Run the Sandcastle and pan to Central park, allow everything to finish loading. Here's the view I used

![image](https://user-images.githubusercontent.com/1495196/74751212-eb733c00-523a-11ea-9994-6a3091631aec.png)

2. Hit `Fly to`.  The flight will start and you'll see styling is half-changed. Hit cancel and the unchanged tiles will not update to the new style

![image](https://user-images.githubusercontent.com/1495196/74751373-24abac00-523b-11ea-87f1-2b18800c8626.png)

The demo has requestRender set, but I confirmed it happens in both cases.
